controller/src/main/java/org/jboss/as/controller/descriptions/ModelDescriptionConstants.java

```diff
@@ -184,6 +184,7 @@
     public static final String FAILURE_DESCRIPTION = "failure-description";
     public static final String FILE = "file";
     public static final String FILE_HANDLER = "file-handler";
+    public static final String FILE_LENGTH = "file-length";
```

Comment:
LogFileResourceDefinition uses file-size; we should be consistent. Using file-length is closer to the java api but since we've already gone with file-size lets stick with it.